tests/pages/pocket/base.py

```diff
@@ -76,11 +76,11 @@ def is_mobile_menu_closed(self):
 
         @property
         def is_mobile_menu_open_button_displayed(self):
-            return self.mobile_menu_open_button
+            return self.is_element_displayed(*self._mobile_menu_open_btn_locator)
 
         @property
         def is_mobile_menu_close_button_displayed(self):
-            return self.mobile_menu_close_button
+            return self.is_element_displayed(*self._mobile_menu_close_btn_locator)
```

Comment:
I think we might see some intermittent test failures here, as `is_mobile_menu_open` can be true before the animation finishes, at which point `is_mobile_menu_close_button_displayed` might still be false. If memory serves me correctly, this is why we added the `transitionend` event originally.

Perhaps adding an explicit wait for `is_mobile_menu_close_button_displayed` inside of `is_mobile_menu_open` might help?

Reply:
good thinking, will do